test/e2e/opm_test.go

```diff
@@ -485,16 +485,28 @@ var _ = Describe("opm", func() {
 	}
 
 	Context("using docker", func() {
+		cmd := exec.Command("docker")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			GinkgoT().Logf("container tool docker not found - skipping docker-based opm e2e tests: (%s) %v", string(out), err)
+			return
+		}
```

Comment:
Hmm yeah, I guess this technically problematic as we're running its CombinedOutput and then Run methods and the command structure doesn't allow being re-used. I'm just going to redirect the stderr pipe to os.Stderr, which would result in the following messages being produced:

```go
time="2021-02-23T12:55:04-05:00" level=error msg="cannot mkdir /run/user/1000/snap.go/libpod: mkdir /run/user/1000/snap.go/libpod: no such file or directory"
container tool podman not found - skipping podman-based opm e2e tests: exit status 1
```